### ISSUE TITLE
Nehkrahqeth from Wyrmstooth causes CTD with certain followers.

### ISSUE THREAD

**Issue by GrandAdmiralVeers** [ForgottenGlory/Living-Skyrim-3#266]
**If you are reporting a crash to desktop, please attach your NET Script Framework crash log. This can be found in `[LS Install folder]\mods\Files Generated during playsessions launched with SKSE\NetScriptFramework\Crash`

If possible, please also attach a copy of your most recent save before the issue occurred.**

[Crash_2021_3_19_0-35-20.txt](https://github.com/ForgottenGlory/Living-Skyrim-3/files/6174291/Crash_2021_3_19_0-35-20.txt)
[Crash_2021_3_19_0-41-45.txt](https://github.com/ForgottenGlory/Living-Skyrim-3/files/6174292/Crash_2021_3_19_0-41-45.txt)
[Crash_2021_3_19_0-52-34.txt](https://github.com/ForgottenGlory/Living-Skyrim-3/files/6174293/Crash_2021_3_19_0-52-34.txt)
[Crash_2021_3_19_0-59-26.txt](https://github.com/ForgottenGlory/Living-Skyrim-3/files/6174294/Crash_2021_3_19_0-59-26.txt)
[Crash_2021_3_19_1-10-8.txt](https://github.com/ForgottenGlory/Living-Skyrim-3/files/6174295/Crash_2021_3_19_1-10-8.txt)
[Crash_2021_3_19_1-24-2.txt](https://github.com/ForgottenGlory/Living-Skyrim-3/files/6174296/Crash_2021_3_19_1-24-2.txt)
[Crash_2021_3_19_17-16-14.txt](https://github.com/ForgottenGlory/Living-Skyrim-3/files/6174297/Crash_2021_3_19_17-16-14.txt)

[Nehkrahqeth CTD File.zip](https://github.com/ForgottenGlory/Living-Skyrim-3/files/6174326/Nehkrahqeth.CTD.File.zip)

**LS Version**
LS 3.0.1

**Describe the bug**
CTD when approaching Nehkrahqeth with certain followers, possibly Toccata or Elisif.

**To Reproduce**
Approach the dragon Nehkrahqeth (first dragon hiding in a swamp on the path in Wyrmstooth) with my party in the save file. 

If you log in to my save file, go down the hill just a little and into the swamp to your right. The dragon will rise up out of it, and the game will CTD after just a few seconds of fighting.

If you tell the followers to wait at the top of the hill and engage the dragon alone, there is no CTD.

**Expected behavior**
My followers should be able to fight Nehkrahqeth.

**Screenshots**
![nehkrahqeth_ctd](https://user-images.githubusercontent.com/80943042/111847534-ff262180-88d6-11eb-95a7-522f7e3bbd6d.png)

**Additional context**
My party has five followers, so I was not able to test which specific followers or which combination causes this crash. However--Elisif and Toccata are the only ones showing up consistently in the crash log, and when I leave all followers behind, I am able to fight without a CTD.

**Comment by WHISKYJVCK** [comment]
I've experienced a similar CTD, this time at the final battle with Alduin. I also have Toccata in my party. Including the logs and the most recent save file.

**Logs**
- [Crash_2021_3_20_12-42-36.txt](https://github.com/ForgottenGlory/Living-Skyrim-3/files/6176026/Crash_2021_3_20_12-42-36.txt)
- [Crash_2021_3_20_13-7-59.txt](https://github.com/ForgottenGlory/Living-Skyrim-3/files/6176027/Crash_2021_3_20_13-7-59.txt)

**Save File**
- [Autosave2_E1B8D195_0_48616B616E_Tamriel_011726_20210320164104_17_1.zip](https://github.com/ForgottenGlory/Living-Skyrim-3/files/6176030/Autosave2_E1B8D195_0_48616B616E_Tamriel_011726_20210320164104_17_1.zip)

**Comment by celadoneiron** [comment]
This is from a bug with Odin's frost spell slow effect. I've reported that bug in Enai's discord and hopefully there'll be an official fix.

The very short version is that if you have two projectiles that will apply the frost slow effect to their target, and they get too close to one another, it triggers a CTD 100% of the time. Dual-casting will do it if you aim at something that's the right distance away for the projectiles to converge on one another before they hit the target, but it also happens with projectiles from separate origins as well.

In GrandAdmiralVeers' case, I think that it's happening because Toccata and Elsif both use the Ice Spike spell and they're aiming at the same target, but that target has high mobility, so they're hitting each others' spells. Captain Motherfaker Davy Jones experienced similar crashes when going into Northwatch Keep with Toccata, specifically when facing off against a frost mage - I'd be willing to bet that Toccata and the mage cast the same spell at each other and the projectiles met and that's what caused the crash.

If we want a temporary patch, overwriting Odin's version of FrostSlowFFAimed (000B729F) and FrostSlowConcAimed (000B729D) with the version from Vokrii stopped the crashes from occurring during my testing.

In the meantime, I suggest not dual-casting ice spells, and removing your followers' access to ice spells, including staves and scrolls, if you can. I know Toccata's Ice Spike can be disabled in her MCM.

**tl;dr: Don't cross the streams.**

**Comment by celadoneiron** [comment]
Okay now I'm big grumpy because I tested this in my non-LS3 bugtesting profile with no mods but Odin and Vokrii and it _was_ doing the same thing, and now it's not, even though I haven't (as far as I know) changed anything. So... damn. I've removed my bug report from Enai's discord for now because I can no longer confirm it's isolated to Odin rather than a result of an interaction in the list, despite my earlier testing.

Edit: it's still 100% reproduceable in LS3 for me, it's just the non-LS3 thing I can't confirm.

**Comment by ForgottenGlory** [comment]
Fixed 3.1.0
